versions/wmde12.env

```diff
@@ -1,34 +1,34 @@
 WMDE_RELEASE_VERSION=wmde.12
-WIKIBASE_BRANCH_NAME=REL1_39
-MEDIAWIKI_BRANCH_NAME=REL1_39
+WIKIBASE_BRANCH_NAME=REL1_38
```

Comment:
This is all in https://github.com/wmde/wikibase-release-pipeline/pull/415
It looks like this PR should be targeting that branch, not master
Then we wouldn't be seeing this file change also in this PR
https://stackoverflow.com/questions/24159036/how-to-modify-a-pull-request-on-github-to-change-target-branch-to-merge-into

Reply:
resolved by later changes